src/Mod/Sketcher/Gui/TaskSketcherElements.h

```diff
@@ -177,6 +177,16 @@ protected Q_SLOTS:
     void onItemHovered(QListWidgetItem *);
 };
 
+class FilterList : public QListWidget

```

Comment:
In line 72 above,
this:
```        , GeometryType(geometryType)```
should read
```        , GeometryType(std::move(geometryType))```
or alternatively, the parameter should have been passed as const reference.

I did not see this in the previous review (although there is a clear warning from the built-in checks).

It goes like this: Either you pass by const &, so you do not make any copy, just bind to the previous value, and then you do copy when you assign, or you do make a copy, but then you move. Otherwise, like the code is today, there are two copies being made.

Reply:
Got it